R/get_branches.R

```diff
@@ -52,13 +51,14 @@ get_branch_info <- function(package = "pipapi", branch = NULL) {
     }
     branches <- branch
   }
-  out <- purrr::map_df(branches, \(x) {
+  out <- lapply(branches, \(x) {
     dat <- latest_commit_for_branch(package, x)
-    dat$commit$author
-  })
-  res <- cbind(package, branch_name = branches, out) %>%
-    dplyr::rename(last_update_time = "date", last_commit_author_name = "name") %>%
-    dplyr::select(-"email")
+    data.frame(dat$commit$author)
+  }) |> rowbind()
+
+  res <- cbind(package, branch_name = branches, out) |>
```

Comment:
Check functions `get_vars()` and `add_vars()` from collapse. They are way faster.

Reply:
Wow, they really are. It is a bit convoluted but faster. 
```
microbenchmark::microbenchmark(
cbind = cbind(num = 1:32, LET = "A", mtcars),
collapse = add_vars(mtcars, num = 1:32, LET = rep("A", nrow(mtcars)), pos = "front")
)

Unit: microseconds
     expr   min     lq    mean median     uq   max neval
    cbind 200.4 205.80 226.025  208.3 215.00 534.4   100
 collapse  23.9  26.85  33.299   33.5  35.15 104.6   100
```